packages/babel-plugin/src/css-prop/__tests__/object-literal.test.tsx

```diff
@@ -729,4 +729,20 @@ describe('css prop object literal', () => {
 
     expect(actual).toInclude('{color:red}');
   });
+
+  it('should retain keys for mapped react components', () => {
+    const actual = transform(`
+        import '@compiled/react';
+
+        ['foo', 'bar'].map((str) => (
+          <div key={str} css={{ backgroundColor: 'blue' }}>
+            {str}
+          </div>
+        ));
+      `);
+
+    console.log(actual);
```

Comment:
You shouldn't need to log the actual here, since jest will output it for you when there is a mismatch. I presume this is lingering code though

Reply:
yeah forgot we don't have an eslint rule to disallow this :s